src/a_star_search/core.clj

```diff
@@ -1,32 +1,34 @@
 (ns a-star-search.core
   (:require [search.algorithms :as alg]
-            [problems.n-puzzle :as np])
+            [problems.hanoi :as np])
```

Comment:
`np` made sense as an abbreviate for N-puzzle, but doesn't make sense for Towers of Hanoi; you probably should have updated that.